### DISCUSSION TITLE
Flex Runtime: Plugins Dashboard. Bulk plugins interaction questions

### DISCUSSION THREAD

**Discussion by Porlin** [twilio-labs/paste#590] [General] [answered]
Hi Design team,

I have spoken with Sarah Li on some feedback for interactions on a pilot initiative for Flex Runtime: Plugins Dashboard. The flow for review is the ability to deploy bulk plugins.

**There are a few interactions I would like the design systems guidance on:**

- Link style button interaction (In the mockups, on rollover it highlights to blue, on click it turns dark blue)
- Badges/labels/tags (What is the common time for the badges/labels to expire?) In the examples I have 'Update available' , 'Pending release'. 'Latest version', 'Active' labels
- Patterns for disabled buttons. In the click-through the 'Pending release' button is only active when there is a saved change inside (think about buying shopping cart, e-commerce check out process ) otherwise it's in a disabled state.
- Modal: The title may be confusing in the last modal and the buttons don't really reflect what it's asking. What is your recommendation here?
- Accordion usage in the 'Releases' page
- Any other things that jump out for you/concerns?

Thank you

**Comment by serifluous** [comment, marked as answer]
Thanks, Porlin! ~~We'll take a look at it during our crit time on Tuesday and get back to you.~~ Will be covered in our next office hours!
